site/docs/get-started/quickstart.md

```diff
@@ -108,28 +108,76 @@ Run the following to install vault and the plugin in development mode:
     kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s
     ```
 
-## Install control plane
+## Install data catalog
 
-??? tip "Install latest development version from GitHub"
+Fybrik requires a data catalog in which to store metadata related to data assets. Currently, two data catalogs are supported:
 
-    The published Helm charts are only available for released versions. 
-    To install the `dev` version install the charts from the source code.
-    For example:
-    ```bash
-    git clone https://github.com/fybrik/fybrik.git
-    cd fybrik
-    helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
-    helm install fybrik charts/fybrik --set global.tag=master -n fybrik-system --wait
-    ```
+ 1. [katalog](https://fybrik.io/dev/reference/katalog/): a data catalog that is included in Fybrik for evaluation purposes.
+ 2. [OpenMetadata](https://open-metadata.org/): An end-to-end metadata management solution that includes data discovery, governance, data quality, observability, and people collaboration.
 
-The control plane includes a `manager` service that connects to a data catalog and to a policy manager. 
-Install the Fybrik release with a built-in data catalog and with [Open Policy Agent](https://www.openpolicyagent.org) as the policy manager:
+No steps are required to install the katalog data catalog, as it is installed by default.
 
+If you prefer to use OpenMetadata, you can either use an existing deployment, or run the following commands to deploy OpenMetadata in kubernetes:
 ```bash
-helm install fybrik-crd fybrik-charts/fybrik-crd -n fybrik-system {{ fybrikVersionFlag }} --wait
-helm install fybrik fybrik-charts/fybrik -n fybrik-system {{ fybrikVersionFlag }}  --wait
+git clone https://github.com/fybrik/fybrik.git
+cd fybrik/third_party/openmetadata/
+make
```

Comment:
Thanks Sima. I will write a script to install OM, so that cloning of the repo would not be necessary.
This script will send API requests to OM, to create all of the table custom properties that we need for fybrik (such as geography)

Reply:
@cdoron please see how other components from the third party are installed in the quick start page; like datashim in https://fybrik.io/v1.1/samples/notebook-write/ and cert-manager...

Reply:
The OpenMetadata installation combines `kubectl apply`-s with helm installations. This is a bit different from both datashim and cert-manager

Reply:
OK, what I meant is that cloning of the fybrik repo can be avoided (datashim is also deployed with `kubectl apply -f `) ... Also I am not sure if pv.yaml is needed for deployment on openshift...

Reply:
Installing OM no longer requires the cloning of the `fybrik` repository.
The documentation was updated to reflect that:
```
curl https://raw.githubusercontent.com/fybrik/fybrik/master/third_party/openmetadata/install_OM.sh | bash -
```